test/e2e/hostedcluster_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Red Hat, Inc.
```

Comment:
@tphee can we rename this one to `hypershift_as_managed_clsuter_test.go`

since hostedcluster is a concept from hypershift... I feel we should expose hypershift over here instead of hostedcluster.

Reply:
I have renamed the test